pagopa/io-app
# feat: [IOBP-30] Add manual code input for IDPay payment authorization

### Depends on https://github.com/pagopa/io-app/pull/4654

## Short description
This PR adds the ability to start an IDPay payment authorization via manual transaction code input.
This PR includes only the happy path of the manual input flow. 

https://github.com/pagopa/io-app/assets/6160324/ec492508-1687-48f8-a83f-6eb8ef4f8705


## List of changes proposed in this pull request
- Added `IDPayPaymentCodeInputScreen` UI
- Added debug UI for `IDPayPaymentAuthorizationScreen` and `IDPayPaymentResultScreen`

## How to test
Tests should solve correctly. 
With the `io-dev-server` (https://github.com/pagopa/io-dev-api-server/pull/260), you can use transaction code `00000001` to test the authorization flow. With `00000002` you should expect an error.
